contributing guide.md

```diff
@@ -0,0 +1,233 @@
+# Introduction to contributing
+Contributions are always welcome, no matter how large or small. Before contributing, please read the [code of conduct](https://docs.opensauced.pizza/contributing/code-of-conduct/) and follow the directions below:
+
+## Recommended Communication Style
+1. Always leave screenshots for visual changes
+2. Always leave a detailed description in the Pull Request. Leave nothing ambiguous for the viewer(s).
+3. Always review your code first. Be sure to run the project locally and test it before asking for a review.
+4. Always communicate in the GitHub repository. Whether it is in the issue or the pull request, keeping the lines of communication open and visible to everyone on the team helps everyone around you.
+
+## Setup
+1. [Fork](https://docs.github.com/en/get-started/quickstart/fork-a-repo) a repository where you are working on [OpenSauced](github.com/open-sauced) to your own GitHub account.
+2. Clone the forked repository to your local machine.
+3. Check the README of the project to determine how to run the project locally. For many of our repositories, you'll find the directions below work for running the project locally.
+4. Run `npm ci` to install the dependencies and set up the project.
```

Comment:
This isn't the current setup for this project, so let's make the updates to this section. You can keep steps 1 and 2. Here's what should follow.

There are two ways to run this project locally:
1. Use the Go Live feature in VS Code.
2. Install the docsify CLI. 

### Docsify CLI
1. In your terminal, install the CLI, `npm i docsify-cli -g`
2. Once the CLI is installed, run `docsify serve`

Once you've done this, you should be able to navigate to the docs site locally.

Reply:
Ok, I see this is below. So there are two options here, you can mention it here and link to that section, or you can move the directions up.